web/src/utils/date/date.test.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {addDays, subDays, addHours} from 'date-fns';
+import {formatDeadlineFromNowText} from 'utils/date';
+
+describe('formatDeadlineFromNowText', () => {
+  test('it should format deadline text accurately', () => {
```

Comment:
it seems we should use it() with describe()?

Reply:
@ifhuang They are actually the same (`'it` is an alias of `test`), depending on convention I guess. Refer: https://jestjs.io/docs/en/api.html#testname-fn-timeout